src/autoscaler/api/policyvalidator/policy_validator_test.go

```diff
@@ -0,0 +1,2344 @@
+						"adjustment": "+1"
+					}]
+				}`
+				})
+				It("should fail", func() {
+					Expect(valid).To(BeFalse())
+					Expect(errResult).To(Equal(&[]PolicyValidationErrors{
+						{
+							Context:     "(root).scaling_rules.0.breach_duration_secs",
+							Description: "Must be greater than or equal to 60",
+						},
+					}))
+				})
+			})
+
+			Context("when breach_duration_secs is greater than 3600", func() {
+				BeforeEach(func() {
+					policyString = `{
+					"instance_max_count":4,
+					"instance_min_count":1,
+					"scaling_rules":[
+					{
+						"metric_type":"memoryutil",
+						"breach_duration_secs": 55000,
+						"threshold":90,
+						"operator":">=",
+						"cool_down_secs":300,
+						"adjustment": "+1"
+					}]
+				}`
+				})
+				It("should fail", func() {
+					Expect(valid).To(BeFalse())
+					Expect(errResult).To(Equal(&[]PolicyValidationErrors{
+						{
+							Context:     "(root).scaling_rules.0.breach_duration_secs",
+							Description: "Must be less than or equal to 3600",
+						},
+					}))
+				})
+			})
+
+			Context("when cool_down_secs is missing", func() {
+				BeforeEach(func() {
+					policyString = `{
+					"instance_max_count":4,
+					"instance_min_count":1,
+					"scaling_rules":[
+					{
+						"metric_type":"memoryutil",
+						"breach_duration_secs": 300,
+						"threshold":90,
+						"operator":">=",
+						"adjustment": "+1"
+					}]
+				}`
+				})
+				It("should succeed", func() {
+					Expect(valid).To(BeTrue())
+				})
+			})
+
+			Context("when cool_down_secs is less than 60", func() {
+				BeforeEach(func() {
+					policyString = `{
+					"instance_max_count":4,
+					"instance_min_count":1,
+					"scaling_rules":[
+					{
+						"metric_type":"memoryutil",
+						"breach_duration_secs": 300,
+						"threshold":90,
+						"operator":">=",
+						"cool_down_secs":5,
+						"adjustment": "+1"
+					}]
+				}`
+				})
+				It("should fail", func() {
+					Expect(valid).To(BeFalse())
+					Expect(errResult).To(Equal(&[]PolicyValidationErrors{
+						{
+							Context:     "(root).scaling_rules.0.cool_down_secs",
+							Description: "Must be greater than or equal to 60",
+						},
+					}))
+				})
+			})
+
+			Context("when cool_down_secs is greater than 3600", func() {
+				BeforeEach(func() {
+					policyString = `{
+					"instance_max_count":4,
+					"instance_min_count":1,
+					"scaling_rules":[
+					{
+						"metric_type":"memoryutil",
+						"breach_duration_secs": 300,
+						"threshold":90,
+						"operator":">=",
+						"cool_down_secs":55000,
+						"adjustment": "+1"
+					}]
+				}`
+				})
+				It("should fail", func() {
+					Expect(valid).To(BeFalse())
+					Expect(errResult).To(Equal(&[]PolicyValidationErrors{
+						{
+							Context:     "(root).scaling_rules.0.cool_down_secs",
+							Description: "Must be less than or equal to 3600",
+						},
+					}))
+				})
+			})
+		})
+		Context("Schedules", func() {
+
+			Context("when timezone is missing", func() {
```

Comment:
how about adding another test case to check whether the timezone value is valid?